### ISSUE TITLE
git fetch hangs in powershell task

### ISSUE THREAD

**Issue by gforceg** [Microsoft/azure-pipelines-tasks#8755]
## Environment

TFS On-Prem Version 16.131.28106.2
Private Agent Version 2.131.0 hosted on Windows Server 2012 R2

## Issue Description

During the CI build I publish a build artifact of my source code (including .git folder) so that I can use it for ef migrations in the release stage.

During the release stage I run `git fetch` within the working directory of the source code using an inline powershell script. All logging at this point stops and the release stays in "In Progress" state indefinitely.

If I remote into the working directory and run `git fetch`  the command returns immediately.

I can recreate the issue with a single task release definition that runs the following inline powershell script:

```ps1
echo "executing git fetch"
git fetch
```

### Task logs

I cannot provide task logs but the last log is an echo command that immediately precedes the git fetch command.

### Error logs

There is no error, the release remains in progress indefinitely.

**Comment by gforceg** [comment]
I can now recreate the issue in remote powershell session by logging in under the service account used by the agent.
I ran `git fetch -v` and I see no output and I can't get the cmd prompt back without hitting ctrl + c.

I ran git --version
git version 2.12.2.windows.2

I logged in as the svc acct using RDP and followed the same steps and a user name / password prompt popped up in a Window. So it seems to be a configuration issue.

Still, it seems wonky that an agent not running in interactive mode should get a pop-up that is impossible to interact with.

Closing this issue as a configuration issue.
